src/panels/lovelace/editor/hui-dialog-edit-card.ts

```diff
@@ -72,157 +69,151 @@ export class HuiDialogEditCard extends LitElement {
         paper-dialog {
           width: 650px;
         }
-        .element-editor {
-          margin-bottom: 16px;
+        .center {
+          margin-left: auto;
+          margin-right: auto;
+        }
+        paper-button paper-spinner {
+          width: 14px;
+          height: 14px;
+          margin-right: 20px;
+        }
+        paper-spinner {
+          display: none;
+        }
+        paper-spinner[active] {
+          display: block;
+        }
+        .hidden {
+          display: none;
         }
       </style>
       <paper-dialog with-backdrop>
-        <h2>Card Configuration</h2>
+        <h2>
```

Comment:
This is becoming quite messy because we're having 3 times a turnery statement on the exact same logic. To make it easier, just extract those into methods.

Preferably I would want to put it in a new custom element but that probably won't work well with the CSS of `paper-dialog`.

```html
<paper-dialog with-backdrop>
  ${this._cardConfig!.id ? this._renderEditor() : this._renderMigrator()}
</paper-dialog>
```